bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/skeleton/factory/Phase1FolderFactory.scala

```diff
@@ -155,5 +157,18 @@ object F2LongslitPartitioner extends Partitioner {
   }).getOrElse(3)
 }
 
-
-
+// IF          R < 6.5 INCLUDE {1}
+// ELIF 6.5 <= R < 10  INCLUDE {2}
+// ELIF 10  <= R < 21  INCLUDE {3}
+// ELIF 21  <= R       INCLUDE {4}
+// ELSE INCLUDE {1},{2},{3},{4} # No magnitude given so include all
+
+object GracesPartitioner extends Partitioner {
+  import edu.gemini.shared.skyobject.Magnitude.Band.R
+  def bucket(t:SPTarget):Int = Option(t.getTarget.getMagnitude(R).getOrNull).map(_.getBrightness).map {R =>
```

Comment:
Yeah I can never remember the right import. But since you caught me I'll fix it ;-)